receiver/filereceiver/go.mod

```diff
@@ -0,0 +1,40 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/receiver/filereceiver
+
+go 1.18
+
+require (
+	github.com/stretchr/testify v1.8.1
+	go.opentelemetry.io/collector v0.67.1-0.20221216200611-892d07dffbb8
```

Comment:
I believe there is a more recent version that this will need to be updated to.

Reply:
Will update, thank you.